packages/components/fields/time-field/src/time-field.tsx

```diff
@@ -0,0 +1,262 @@
+import {
+  Component,
+  isValidElement,
+  type KeyboardEvent,
+  type MouseEvent,
+  type ReactElement,
+  type ReactNode,
+} from 'react';
+import {
+  filterDataAttributes,
+  createSequentialId,
+  getFieldId,
+  warning,
+} from '@commercetools-uikit/utils';
+import Constraints from '@commercetools-uikit/constraints';
+import Stack from '@commercetools-uikit/spacings-stack';
+import FieldLabel from '@commercetools-uikit/field-label';
+import TimeInput from '@commercetools-uikit/time-input';
+import FieldErrors from '@commercetools-uikit/field-errors';
+
+type TEvent = {
+  target: {
+    id?: string;
+    name?: string;
+    value?: unknown;
+  };
+  persist?: () => void;
+};
+
+type TFieldErrors = Record<string, boolean>;
+
+type TTimeFieldProps = {
+  // TimeField
+  /**
+   * Used as HTML id property. An id is auto-generated when it is not specified.
+   */
+  id: string;
```

Comment:
I think this should be optional

Reply:
you are so right

Fixed in 029c99d7583c8d4d2b365acea64cbce037dfb360

Thanks 👍